automation/test-linux.sh

```diff
@@ -0,0 +1,126 @@
+#!/bin/bash
+
+set -ex
+
+template_name=$1
+namespace="kubevirt"
+
+image_url=""
+#set secret_ref only for rhel OSes
+secret_ref=""
+if [[ $TARGET =~ rhel.* ]]; then
+  image_url="docker://quay.io/openshift-cnv/ci-common-templates-images:${TARGET}"
+  secret_ref="secretRef: common-templates-container-disk-puller"
+else
+  image_url="docker://quay.io/kubevirt/common-templates:${TARGET}"
+fi;
+
+oc apply -n $namespace -f - <<EOF
+apiVersion: cdi.kubevirt.io/v1beta1
+kind: DataVolume
+metadata:
+  name: ${TARGET}-datavolume-original
+spec:
+  source:
+    registry:
+      url: "${image_url}"
+      ${secret_ref}
+  pvc:
+    accessModes:
+      - ReadWriteOnce
+    resources:
+      requests:
+        storage: 30Gi
+EOF
+
+timeout=600
+sample=10
+
+oc wait --for=condition=Ready --timeout=${timeout}s dv/${TARGET}-datavolume-original -n $namespace
```

Comment:
Should the tests fail, if these `oc wait` commands time out?

Reply:
in this case yes, because otherwise it will not import the base image